src/themes/button-styles.ts

```diff
@@ -33,6 +54,9 @@ export const buttonStyles = {
 
     left: 'mr-2 -ml-1',
     right: 'ml-2 -mr-1',
+
+    // Missing style? Because apparently its used in use-button
```

Comment:
please verify if these styles are missing from the theme or is it intentional that they were not declared here?